src/lib/features/integration-events/integration-events-store.ts

```diff
@@ -41,11 +41,21 @@ export class IntegrationEventsStore extends CRUDStore<
                 qb.select('id')
                     .from(this.tableName)
                     .whereRaw(`created_at >= now() - INTERVAL '2 hours'`)
-                    .orderBy('created_at', 'desc')
+                    .orderBy('id', 'desc')
                     .limit(100);
             })
+            .with('latest_per_integration', (qb) => {
+                qb.select(this.db.raw('DISTINCT ON (integration_id) id'))
```

Comment:
Should we add something like max per id or something? This query might be expensive. I'm not sure but maybe  you already considered this

Reply:
I don't expect this to be expensive but you may be thinking of something that I missed. 
Let me know if this is better: https://github.com/Unleash/unleash/pull/7652/commits/272cd0fb4094cc29f3837e9527adca4b1c37f86d